setup.py

```diff
@@ -93,7 +93,6 @@ def setup(*args, **kwargs):
         "gsd>=2.0",
         "garnett>=0.7.1",
         "matplotlib>=3.0",
-        "MDAnalysis>=0.20.1",
```

Comment:
Is MDAnalysis a soft dependency now?

Reply:
MDAnalysis is not required for freud, but we have historically used it when testing the package because we provide interoperability (freud can extract periodic boxes and positions from MDAnalysis timesteps). I removed MDAnalysis as a test requirement because it has been really difficult for us to find versions that work on all the platforms and Python/OS versions that freud supports. Because of the challenges we've faced both in this PR and historically (there are workarounds in our existing deploy scripts), we have to drop it as a testing dependency. The MDAnalysis package won't support Python 3.9 until version 2.0, which isn't released yet, and it also has had a lot of issues on Mac (no wheels are available, and building from source fails). You can see a number of related issues on this search page: https://github.com/MDAnalysis/mdanalysis/issues?q=3.9